addons/website/static/src/js/content/menu.js

```diff
@@ -396,6 +444,44 @@ publicWidget.registry.FixedHeader = BaseAnimatedHeader.extend({
                     this.hiddenOnScrollEl.style.paddingBlock = "";
                 }
             }
+            if (!this.fixedHeader && this.dropdownClickedEl) {
+                const dropdown = Dropdown.getOrCreateInstance(this.dropdownClickedEl);
+                dropdown.show();
+                this.dropdownClickedEl = null;
+            }
+        }
+    },
+    /**
+     * Called when a dropdown within 'this.hiddenOnScrollEl' is clicked.
+     *
+     * @private
+     * @param {Event} ev
+     */
+    _onDropdownShow(ev) {
+        // If a dropdown inside the element 'this.hiddenOnScrollEl' is clicked
+        // while the header is fixed, we need to scroll the page up so that the
+        // 'this.hiddenOnScrollEl' element is no longer overflow hidden. Without
+        // this, the dropdown would be invisible.
+        if (this.fixedHeader) {
+            ev.preventDefault();
+            this.scrollableEl.scrollTo({ top: 0, behavior: "smooth" });
```

Comment:
Isn't there the same risk here? If I focus a dropdown in the `hiddenOnScrollEl, then I scroll down and press enter. The dropdown could break the header no ?

Reply:
> Isn't there the same risk here? If I focus a dropdown in the `hiddenOnScrollEl, then I scroll down and press enter. The dropdown could break the header no ?

no because we added `ev.preventDefault();` here and the dropdown is only shown later in `_updateHeaderOnScroll` once the header is no longer fixed

Reply:
Ok thanks! The diff looks good to me, I tested and it's working good. Could you just add a test that setup a header `Menu - Sales 2` and check that the search result and the user dropdown are visible when we use it.